### PR TITLE
Add fetch-mock mocks for noembed.com thumbnail-fetching for dashboard

### PR DESCRIPTION
**Pull request by ianmcorvidae**
This is a bit ugly, recommendations welcome. But it does get the dashboard back to working.

### CODE REVIEW

**Comment by ianmcorvidae** [comment]
PS the bodies I've put in these mocks are just from doing the actual requests with `curl`

File: src/components/dashboard/dashboardItem/ItemBase.js [hunk 1]
```diff
@@ -212,7 +212,7 @@ export const DashboardVideoItem = ({ item }) => {
         >
             <ReactPlayer
                 url={item.getLinkTarget()}
-                light={true}
+                light={item.content.thumbnailUrl || true}
```
Comment (ianmcorvidae) [inline src/components/dashboard/dashboardItem/ItemBase.js:215]: This change means that if the data that comes from dashboard-aggregator includes a `thumbnailUrl` (as I added manually to one of the videos in the data.json below) then noembed won't be called at all.

**Review by ash3rz — APPROVED** [review]
LGTM 👍

**Comment by ianmcorvidae** [comment]
Cool, I'll merge this for now. If I update dashboard-aggregator to provide the thumbnails, I can update the data file here to reflect that too (though I might leave one that uses noembed so that keeps being tested, little though it is).

**Pull request merged**